lib/coverband/memory_cache_store.rb

```diff
@@ -32,12 +32,7 @@ def filter(files)
         #first time we see a file, we pre-init the in memory cache to whatever is in store(redis)
         line_cache = files_cache[file] ||= Set.new(store.covered_lines_for_file(file))
         lines.reject! do |line|
-          if line_cache.include? line
-            true
-          else
-            line_cache << line
-            false
-          end
+          line_cache.include?(line) ? true : (line_cache << line and false)
```

Comment:
I agree, that's why I prefer don't using in this way, but for me 

`line_cache.include?(line) ? true : (line_cache << line and false)` 

is very easy readable.